packages/web/src/pages/search-page-v2/SearchFilters.tsx

```diff
@@ -216,33 +317,134 @@ const BpmFilter = () => {
           anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
           transformOrigin={{ vertical: 'top', horizontal: 'left' }}
         >
-          <Paper mt='s' border='strong' shadow='far'>
+          <Paper mt='s' border='strong' shadow='far' css={{ width: 242 }}>
             <Flex
-              p='s'
+              w='100%'
+              pv='s'
+              gap='s'
               direction='column'
               alignItems='flex-start'
               role='listbox'
             >
-              <Flex direction='column' w='100%' gap='s'>
+              <Flex
+                direction='column'
+                w='100%'
+                ph='s'
+                onClick={(e) => e.stopPropagation()}
```

Comment:
how come this needs to be here? can segmented control be the one to stop propagation?

Reply:
ahh dope! maybe a comment then?